benchmarks/test_hbf.py

```diff
@@ -0,0 +1,47 @@
+import os
+import numpy as np
+import pyccl as ccl
+
+# Set cosmology
+cosmo = ccl.Cosmology(Omega_c=0.25, Omega_b=0.05, Omega_g=0, Omega_k=0,
+                      h=0.7, sigma8=0.8, n_s=0.96, Neff=0, m_nu=0.0,
+                      w0=-1, wa=0, T_CMB=2.7255,
+                      transfer_function='eisenstein_hu')
+# Read data
+dirdat = os.path.dirname(__file__) + '/data/'
+
+# Redshifts
+zs = np.array([0., 0.5, 1.])
+
+
+def test_hbf_tinker10():
+    d_hbf = np.loadtxt(dirdat + 'hbf_tinker10.txt',
+                       unpack=True)
+    mf = ccl.halos.HaloBiasTinker10(cosmo)
+    m = d_hbf[0]
```

Comment:
We only test one mass? Ditto for the ones below.

Reply:
nope, `d_hbf[0]` is the first column of the data file, which contains to a bunch of masses.

Reply:
Ahhhhh great. So confusing!